src/components/Filters/StatefulFilters.tsx

```diff
@@ -150,7 +155,7 @@ export class StatefulFilters extends React.Component<StatefulFiltersProps, State
     }
   };
 
-  filterValueSelected = (value: string) => {
+  filterValueSelected = (_: any, value: any) => {
```

Comment:
Can the `_` be something with a more meaningful name?

Reply:
yep, it can be. However, it is not used and then, the linter jumps and says it should use it. So the only way you can avoid this one is using that underscore.

Reply:
You can actually use `_foo` as the name and the compiler will know it is not supposed to be used.

Reply:
Good point!